generators/profilegen/src/resources/templates/client_template.template

```diff
@@ -0,0 +1,46 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative './module_definition'
+<%- @file_names.each do |file_name| -%>
+require_relative 'modules/<%= file_name %>'
+<%- end -%>
+require_relative '../utils/configurable'
+require_relative '../utils/default'
+
+module Azure::Profiles::<%- if @individual_gem_profile -%><%= @module_name %>::<%- end -%>Management::<%= @profile_name %>
+  #
+  # Client class for the <%= @profile_name %> profile SDK.
+  #
+  class Client
+    include Azure::ARM::Configurable
```

Comment:
❗️ ❗️ 
I am confused on how we ship `Azure::ARM::Configurable` and `Azure::ARM::Default` inside each gem's profile's `utils` folder even though that gems does not have that namespaces defined.

Example:
`module_definition` of azure_mgmt_analysis_services's `Profile_latest` [here](https://github.com/sarangan12/azure-sdk-for-ruby/blob/6ebbce2f4476af9eb19906ded03e9175e724efa6/management/azure_mgmt_analysis_services/lib/profiles/Profile_Latest/module_definition.rb#L9) does not define namesapce `Azure::ARM::Configurable` but getting shipped inside `utils` folder [here](https://github.com/sarangan12/azure-sdk-for-ruby/blob/6ebbce2f4476af9eb19906ded03e9175e724efa6/management/azure_mgmt_analysis_services/lib/profiles/utils/configurable.rb#L7)

Reply:
We can discuss this in person.